torch_patches/dpcpp-v1.5-rc3.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/aten/src/ATen/SparseTensorImpl.cpp b/aten/src/ATen/SparseTensorImpl.cpp
-index a7354047ca..3bf11a646f 100644
+index a735404..3bf11a6 100644
```

Comment:
nit: hash from your git is only 8-letter as opposed to the original 10-letter hash.
Maybe you are using an old git. I'm using v2.23.0